tesseract_core/sdk/tesseract.py

```diff
@@ -62,49 +63,77 @@ class Tesseract:
 
     def __init__(self, url: str) -> None:
         self._spawn_config = None
-        self._project_id = None
-        self._container_id = None
+        self._serve_context = None
         self._lastlog = None
         self._client = HTTPClient(url)
 
+    @classmethod
+    def from_url(cls, url: str) -> Tesseract:
```

Comment:
In principle I'm against having two ways of doing the same thing, but ok for me to keep

Reply:
Hmm yes I think ideally we'd deprecate the standard `__init__` and just use the classmethod constructors. Thoughts?

Reply:
Feels worse -- let's keep as it is now 👍